openlibrary/components/MergeUI.vue

```diff
@@ -51,36 +66,42 @@ export default {
             this.mergeStatus = 'Saving...';
             try {
                 const r = await do_merge(master, dupes, editions_to_move, this.mrid);
-                this.mergeStatus = await r.json();
-
+                this.mergeOutput = await r.json();
                 if (this.mrid) {
                     await update_merge_request(this.mrid, 'approve', this.comment)
                 } else {
                     const workIds = [master.key].concat(Array.from(dupes, item => item.key))
                     await createMergeRequest(workIds)
                 }
             } catch (e) {
-                this.mergeStatus = e.message;
+                this.mergeOutput = e.message;
                 throw e;
             }
+            this.mergeStatus = 'Do Merge';
```

Comment:
It's actually a little confusing since the button becomes green so quickly, and the feedback that it succeeded is further below the page. I think it might result in people clicking the button twice, which would probably cause issues :P Let's set the status to Done
```suggestion
            this.mergeStatus = 'Done';
```